tests/IBFE/explicit_ex4.cpp

```diff
@@ -530,6 +531,31 @@ main(int argc, char** argv)
             other_manager->setPatchHierarchy(patch_hierarchy);
         }
 
+        if (input_db->getBoolWithDefault("test_markers", false))
+        {
+            EigenAlignedVector<IBTK::Point> positions;
+
+            const auto n_nodes = mesh.parallel_n_nodes();
+            for (dof_id_type i = 0; i < n_nodes; ++i)
+            {
+                IBTK::Point X;
+                for (int d = 0; d < NDIM; ++d)
+                {
+                    X[d] = mesh.node_ref(i)(d);
+                }
+                // This is a copy-paste of coordinate_mapping_function() but for IBTK points
+                X[0] += 0.6;
+                X[1] += 0.5;
+#if NDIM == 3
+                X[2] += 0.5;
+#endif
+
+                positions.push_back(X);
+            }
+
+            time_integrator->setMarkers(positions);
+        }
```

Comment:
@jabrown893 You'll be the first user - does something like this make sense for how you plan on using this new feature?